agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -63,7 +63,9 @@ const (
 	// ECSAgentExecConfigDir is the directory where ECS Agent will write the ExecAgent config files to
 	ECSAgentExecConfigDir = ecsAgentExecDepsDir + "/" + ContainerConfigDirName
 	// HostExecConfigDir is the dir where ExecAgents Config files will live
-	HostExecConfigDir = hostExecDepsDir + "/" + ContainerConfigDirName
+	HostExecConfigDir          = hostExecDepsDir + "/" + ContainerConfigDirName
+	ExecAgentLogConfigFileName = "seelog.xml"
```

Comment:
I think we probably want to follow the same naming convention that we do with `execAgentConfigFileNameTemplate` (using the SHA in the name of the file). This file might be confgurable in the future and when that happens we will be ready. Plus it's pretty much the same code that we already use for the config file.

Reply:
after offline discussion, decided it would be useful to use hash for initial launch as well.